src/fontra/views/editor/editor.js

```diff
@@ -2030,6 +2104,259 @@ export class EditorController {
     return { contentElement, warningElement };
   }
 
+  selectionHasLockedGuidelines() {
+    const {
+      guidelineGlyph: guidelineGlyphSelection,
+      guidelineFont: guidelineFontSelection,
+    } = parseSelection(this.sceneController.selection);
+
+    const instance = this.sceneModel.getSelectedPositionedGlyph()?.glyph.instance;
+    for (const guidelineIndex of guidelineGlyphSelection || []) {
+      const guidelineGlyph = instance.guidelines[guidelineIndex];
+      if (guidelineGlyph.locked) {
```

Comment:
Suggest to omit the variable, and just write
```js
      if (instance.guidelines[guidelineIndex].locked) {
        return true;
      }

Reply:
Better suggestion for the whole for loop:
```js
    if (guidelineGlyphSelection?.some((index) => instance.guidelines[index].locked) {
      return true;
    }
```

Reply:
Thanks a lot. I changed it.